ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -65,6 +88,34 @@ data = yaml.load(open(path));
 data["spec"]["mastersSchedulable"] = False;
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
+### Remove DNS Zones (optional)
+if [[ -v IS_XPN ]]; then
+  echo "Removing DNS Zones..."
+  python -c '
+import yaml;
+path = "manifests/cluster-dns-02-config.yml";
+data = yaml.load(open(path));
+del data["spec"]["privateZone"];
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+fi
+
+### Update the cloud-provider manifest ([Shared VPC (XPN)][sharedvpc] only)
+if [[ -v IS_XPN ]]; then
+  echo "Updating the cloud-provider manifest..."
+  sed -i "s/    subnetwork-name.*/    network-project-id = ${HOST_PROJECT}\\n    network-name    = ${HOST_PROJECT_NETWORK_NAME}\\n    subnetwork-name = ${HOST_PROJECT_COMPUTE_SUBNET_NAME}/" manifests/cloud-provider-config.yaml
+fi
+
+### Enable external ingress (optional)
+if [[ -v IS_XPN ]]; then
```

Comment:
this and the previous conditional could all be combined with the first: 1 `if` instead of 3.

Reply:
That is true. However, the associated comments need to line up in order to make auditing this script easier using diff. The additional CPU power is insignificant. Plus, we may need to use these independently if we add additional variants.